packages/rainbowkit/src/components/ConnectModal/ConnectModal.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { describe, expect, it } from 'vitest';
+import { mainnet } from 'wagmi/chains';
+import { renderWithProviders } from '../../../test';
+import { Locale } from '../../locales';
+import { ConnectModal } from './ConnectModal';
+
+describe('<ConnectModal />', () => {
+  const renderHeaderLabelModal = async (locale?: Locale) => {
+    const options = {
+      mock: true,
+      props: {
+        chains: [mainnet],
+        ...(locale ? { locale } : {}),
+      },
+    };
+
+    const { getByTestId } = renderWithProviders(
+      <ConnectModal onClose={() => {}} open={true} />,
+      options,
+    );
+
+    const modal = getByTestId('rk-connect-header-label');
+
+    return modal.textContent;
+  };
+
+  it('Defaults to English without a `locale` prop', async () => {
+    const label = await renderHeaderLabelModal();
+    expect(label).toBe('Connect a Wallet');
+  });
+
+  it("Displays in English for 'en-US'", async () => {
+    const label = await renderHeaderLabelModal('en-US');
+    expect(label).toBe('Connect a Wallet');
+  });
```

Comment:
✨ Tests for `<ConnectModal />`.

I was also thinking of doing like this:
```
i18n.locale = "ru-RU"
const label = await renderHeaderLabelModal('en-US');
expect(label).toBe(i18n.t("keyword"));
```

But will require more code ofc, tried to keep it clean in this case 🤷‍♂️